src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/LocationSchedule.kt

```diff
@@ -0,0 +1,72 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity
+
+import jakarta.persistence.Entity
+import jakarta.persistence.EnumType
+import jakarta.persistence.Enumerated
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+import org.hibernate.Hibernate
+import java.time.LocalDateTime
+import java.time.LocalTime
+
+@Entity
+@Table(name = "location_schedule")
+data class LocationSchedule(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val locationScheduleId: Long = 0,
+
+  @JoinColumn(name = "location_attribute_id")
+  @ManyToOne(fetch = FetchType.LAZY)
+  val locationAttribute: LocationAttribute,
+
+  val startDayOfWeek: Int,
+
+  val endDayOfWeek: Int,
+
+  val startTime: LocalTime,
+
+  val endTime: LocalTime,
+
+  @Enumerated(EnumType.STRING)
+  val locationUsage: LocationUsage,
```

Comment:
location usage and allowed parties is in both entities, can you expand on how they differ?

Reply:
LocationUsage one of (COURT, PROBATION, SHARED, or SCHEDULE).
If SCHEDULE, its usage can alter at different days of the week or times of the day, so LocationUsage at the schedule level, can be COURT, PROBATION or SHARED .. but at different days/times.